lighthouse-core/gather/gatherers/dobetterweb/domstats.js

```diff
@@ -148,8 +148,8 @@ class DOMStats extends Gatherer {
     })()`;
     return passContext.driver.sendCommand('DOM.enable')
       .then(() => passContext.driver.evaluateAsync(expression, {useIsolation: true}))
-      .then(results => passContext.driver.getElementsInDocument().then(allNodes => {
-        results.totalDOMNodes = allNodes.length;
+      .then(results => passContext.driver.getElementsInDocument('body').then(bodyNodes => {
```

Comment:
I don't think this will do what we want it to do. The implementation of `getElementsInDocument` looks like it doesn't accept a querySelector

https://github.com/GoogleChrome/lighthouse/blob/e6dbefa15c07903733a9b1f90dbb9ae393b35cb6/lighthouse-core/gather/driver.js#L1141-L1153

Reply:
Yup, I misleadingly thought I can pick `getElementsInDocument()` from those page-functions...